src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -130,7 +131,7 @@ public ExchangeService(BidderCatalog bidderCatalog, StoredResponseProcessor stor
         this.currencyService = currencyService;
         this.bidResponsePostProcessor = Objects.requireNonNull(bidResponsePostProcessor);
         this.privacyEnforcementService = Objects.requireNonNull(privacyEnforcementService);
-        this.eventsService = eventsService;
+        this.eventsService = Objects.requireNonNull(eventsService);
```

Comment:
is there any reason why there wasn't null check ?

Reply:
I guess it just was missed, seems like EventsService is not a conditional bean therefore it always should be available. Either way, lets keep the null check - it won't harm, @rpanchyk right?

Reply:
Right, EventsService is mandatory. So, need to be checked for null. Thanks.